README.md

```diff
@@ -19,7 +19,7 @@
 
 ![](site/docs/img/Iceberg-logo.png)
 
-[![](https://travis-ci.org/apache/iceberg.svg?branch=master)](https://travis-ci.org/apache/iceberg)
+[![](https://travis-ci.com/apache/iceberg.svg?branch=master)](https://travis-ci.com/apache/iceberg)
```

Comment:
We actually use github actions these days. Maybe we can just remove this entirely? Is there an equivalent for github actions?

Reply:
It looks like there is, let me see if I can update it.

Reply:
I updated this to add the java-ci and python-ci workflows badges.